website/content/en/docs/building-operators/ansible/reference/webhooks.md

```diff
@@ -36,30 +37,19 @@ be hitting the real API server and will not get caching for free.
 ## Deploying the webhook server
 
 To deploy the webhook server as a sidecar alongside your operator, all you need to do is add the container
-specification to your `deploy/operator.yaml`. You may also need to add a volume for mounting in TLS secrets,
+specification to your `config/manager/manager.yaml`. You may also need to add a volume for mounting in TLS secrets,
 as your webhook server is required to have a valid SSL configuration. Below is a sample updated container
 specification that deploys a webhook:
 
 ```yaml
 containers:
-  - name: my-operator
-    # Replace this with the built image name
-    image: "REPLACE_IMAGE"
+  - name: manager
+    image: controller:latest
     imagePullPolicy: "Always"
     volumeMounts:
     - mountPath: /tmp/ansible-operator/runner
       name: runner
     env:
-      - name: WATCH_NAMESPACE
-        valueFrom:
-          fieldRef:
-            fieldPath: metadata.namespace
-      - name: POD_NAME
-        valueFrom:
-          fieldRef:
-            fieldPath: metadata.name
-      - name: OPERATOR_NAME
-        value: "validating-operator"
       - name: ANSIBLE_GATHERING
```

Comment:
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: controller-manager
  namespace: system
  labels:
    control-plane: controller-manager
spec:
  selector:
    matchLabels:
      control-plane: controller-manager
  replicas: 1
  template:
    metadata:
      labels:
        control-plane: controller-manager
    spec:
      containers:
        - name: manager
          args:
            - "--enable-leader-election"
            - "--leader-election-id=memcached-operator"
          image: controller:latest
          volumeMounts:
             - mountPath: /tmp/ansible-operator/runner
                name: runner
          env:
            - name: ANSIBLE_GATHERING
              value: "explicit
      terminationGracePeriodSeconds: 10

```

Reply:
See that it changed.

Reply:
I rewrote this to use a kustomize patch. It makes the section _much_ simpler for readers, I think.